kernel/src/engine/arrow_data.rs

```diff
@@ -339,20 +340,22 @@ mod tests {
         let engine = SyncEngine::new();
         let handler = engine.get_json_handler();
         let json_strings: StringArray = vec![
-            r#"{"protocol": {"minReaderVersion": 3, "minWriterVersion": 7, "readerFeatures": ["rw1"], "writerFeatures": ["rw1", "w2"]}}"#,
+            r#"{"protocol": {"minReaderVersion": 3, "minWriterVersion": 7, "readerFeatures": ["deletionVectors"], "writerFeatures": ["invariants", "appendOnly"]}}"#,
```

Comment:
can we also have unknown table features here? e.g. what happens if we leave a 'rw1' in here